tools/jenkins/run_distribution.sh

```diff
@@ -66,67 +66,61 @@ if [ "$platform" == "linux" ]; then
 elif [ "$platform" == "macos" ]; then
 
   if [ "$dist_channel" == "homebrew" ]; then
-    # system installed homebrew, don't interfere
+    # System installed homebrew, don't interfere. Run this again at the end
+    # to make sure the same formulas were still installed
     brew list -l
```

Comment:
i'd do echo "Formulas installed by system-wide homebrew (BEFORE):"
and echo "Formulas installed by system-wide homebrew (AFTER - should be left intact):"  to make sense of the job logs.
Then you can also get rid of some of the comments as everyone will know what's happening.